Tasks/XamarinTestCloud/XamarinTestCloud.ps1

```diff
@@ -158,7 +158,14 @@ foreach ($ap in $appFiles)
         $argument = "$argument --nunit-xml ""$nunitFileCurrent"""
     }
     Write-Host "Submit $ap to Xamarin Test Cloud."
-    Invoke-Tool -Path $testCloud -Arguments $argument 
+    Invoke-Tool -Path $testCloud -Arguments $argument -OutVariable toolOutput
+    foreach($line in $toolOutput)
+    {
+        if($line -match "https://testcloud.xamarin.com/test/(.+)/")
```

Comment:
You might consider using -imatch instead of -match so that the case-insensitivity is explicit/clearer.